it-IT.Rmd

```diff
@@ -60,32 +60,31 @@ Intimidazione e bullismo includono, ma non sono limitati a:
+Se una o più persone contravvengono al Codice di Comportamento, la Commissione del Codice di Comportamento si ritiene in diritto di agire in qualunque modo al fine di assicurare un ambiente confortevole per tutti i membri della comunità.
 Questo include l'avvertire il presunto colpevole o l'espulsione temporanea/permanente dall'evento e/o dalle piattaforme elettroniche sotto il controllo di Bioconductor.
 
-La Commissione del Codice di Buona Condotta può agire in modo da porre rimedio a qulasiasi cosa progettata per, o con il chiaro impatto di, disturbare un evento o una piattaforma di comunicazione elettronica o da rendere l'ambiente ostile per qualsiasi membro della comunità.
+La Commissione del Codice di Comportamento può agire in modo da porre rimedio a qulasiasi cosa progettata per, o con il chiaro impatto di, disturbare un evento o una piattaforma di comunicazione elettronica o da rendere l'ambiente ostile per qualsiasi membro della comunità.
 
-Ci aspettiamo che ogni membro della comunità di Bioconductor si attenga al Codice di Buona Condotta durante la partecipazione agli eventi di Bioconductor e sulle piattaforme elettroniche online.
+Ci aspettiamo che ogni membro della comunità di Bioconductor si attenga al Codice di Comportamento durante la partecipazione agli eventi di Bioconductor e sulle piattaforme elettroniche online.
 
 ## Segnalazione
 
 Se qualcuno fa sentire te o chiunque altro non al sicuro o non benvenuto, sei pregato di riportarlo il prima possibile. 
 La segnalazione può essere sia in forma anonima che personale. 
-Tutte le segnalazioni saranno valutate dalla Commissione del Codice di Buona Condotta e verranno trattate in maniera confidenziale.
+Tutte le segnalazioni saranno valutate dalla Commissione del Codice di Comportamento e verranno trattate in maniera confidenziale.
 
 #### _Segnalazione per via Elettronica_
 
-Puoi fare una segnalazione anonima o non-anonima al seguente link:
-<https://tinyurl.com/bioccomplaint>.
-Dove è presente un box di testo in formato libero che verrà inviato alla Commissione del Codice di Buona Condotta.
-In alternativa puoi inviare una email alla Commissione del Codice di Buona Condotta (<code-of-conduct@bioconductor.org>).
-Se non ti senti a tuo agio a fare una segnalazione alla Commissione del Codice di Buona Condotta come gruppo, puoi contattare un qualsiasi membro della Commissione del Codice di Buona Condotta tramite email o con un messaggio privato sul canale Slack \"community-bioc\". Per favore includi \"screenshots\"/copie di tutte le conversazioni rilevanti ogni volta possibile (non c'è bisogno di compromettere il tuo anonimato!).
+Puoi fare una segnalazione anonima o non-anonima al seguente link [https://forms.gle/gEWHBWnXvZbEdFsq5](https://forms.gle/gEWHBWnXvZbEdFsq5).
+Dove è presente un box di testo in formato libero che verrà inviato alla Commissione del Codice di Comportamento.
```

Comment:
Here's a non-full sentence. We might need to remove the full stop after the link and start with a lowercase